### ISSUE TITLE
LR Config

### ISSUE THREAD

**Issue by BalasubramanyamEvani** [open-mmlab/mmsegmentation#3008]
Hi,

I don't want to use a lr scheduler. but, if I don't put lr_config in my configurations, it throws an error. How do I resolve this?

MMSegmentation Version: 0.30.0
mmcv version: https://download.openmmlab.com/mmcv/dist/cu101/torch1.8.0/index.html

**Comment by daeunni** [comment]
@BalasubramanyamEvani  Hi, did u solve this problem?

**Comment by BalasubramanyamEvani** [comment]
@daeunni Hey, I think so. I defined an iterbased runner and in the lr config, I added by_epoch=True. 

```python
runner = dict(type="IterBasedRunner", max_iters=max_iters)
lr_config = dict(policy="poly", power=0.9, min_lr=0.0001, by_epoch=True)
```

**Comment by daeunni** [comment]
What does mean by "`by_epoch`"? :0

**Comment by BalasubramanyamEvani** [comment]
I believe "by_epoch" as the name suggests means to apply the policy per epoch rather than per iteration. The answer I shared earlier seems like a hack I guess a better way would be to use ConstantLR with a factor of 1. 

See here: https://github.com/open-mmlab/mmengine/blob/main/mmengine/optim/scheduler/lr_scheduler.py

Let me know what you think @daeunni

**Comment by daeunni** [comment]
I agree with your thought. Ty for sharing :)
